server/src/main/scala/controllers/application/WithDebug.scala

```diff
@@ -26,10 +27,10 @@ trait WithDebug {
     Action.async { _ =>
       val pit = SDate(dateString)
       val persistenceIds = SortedMap(
-        "ACL" -> "actors.ForecastBaseArrivalsActor-forecast-base",
-        "Port Forecast" -> "actors.ForecastPortArrivalsActor-forecast-port",
-        "Cirium Live" -> "actors.LiveBaseArrivalsActor-live-base",
-        "Port Live" -> "actors.LiveArrivalsActor-live",
+        "ACL" -> AclForecastArrivalsActor.persistenceId,
```

Comment:
nice refactor.

Reply:
When I moved the actors into a new package intellij updated all strings with the class path in them! So then I discovered we had those persistence IDs in quite a number of places...